docs/migration/5_0.md

```diff
@@ -40,3 +40,7 @@ under the section `headerDisplayOnly`.
 #### Translations (i18n) changes
 
 - `configurator.addToCart.buttonDisplayOnly` was added to `configurator-common.ts`
+
+#### PromotionsComponent
+
+- Component template was modified to display both a `single` promotion and a `list` of promotions
```

Comment:
@RadhepS can we mention in the doc migration in little bit more details (including specific tags) what changed in compare to previous version? Maybe something like:
"Component template was modified to display for single promotion the description in `p` tag, otherwise if multiple promotions to display them in `ul` list elements."